### PR TITLE
Add regression test for negative margins

### PR DESCRIPTION
**Pull request by asamuzaK**
Regression test for https://github.com/jsdom/cssstyle/issues/222

### CODE REVIEW

File: test/web-platform-tests/to-upstream/css/css-box/negative-margin.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<div id="div"></div>
+
+<script>
+"use strict";
+const div = document.getElementById("div");
+
+test(() => {
+  div.style.marginTop = "-10px";
+  assert_equals(div.style.getPropertyValue("margin-top"), "-10px");
+  div.style.marginTop = null;
```
Comment (domenic) [inline test/web-platform-tests/to-upstream/css/css-box/negative-margin.html]:
```suggestion
test(t => {
  t.add_cleanup(() => { div.style.marginTop = null; });
  div.style.marginTop = "-10px";
  assert_equals(div.style.getPropertyValue("margin-top"), "-10px");
```

here and below. This is best practice to ensure that one test failing does not cause all other tests to fail.

**Review by domenic — CHANGES_REQUESTED** [review]

File: test/web-platform-tests/to-upstream/css/css-box/negative-margin.html (hunk 1 above)
Comment (asamuzaK) [reply test/web-platform-tests/to-upstream/css/css-box/negative-margin.html]: Done

**Review by domenic — APPROVED** [review]
